test/tools/jsontests/TransactionTests.cpp

```diff
@@ -159,115 +158,76 @@ json_spirit::mValue doTransactionTests(json_spirit::mValue const& _input, bool _
-{
-	dev::test::executeTests("ttTransactionTest", "/TransactionTests/Metropolis", "/TransactionTestsFiller/Metropolis", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttMetropolisTests)
-{
-	dev::test::executeTests("ttMetropolisTest", "/TransactionTests/Metropolis", "/TransactionTestsFiller/Metropolis", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttTransactionTestEip155VitaliksTests)
-{
-	dev::test::executeTests("ttTransactionTestEip155VitaliksTests", "/TransactionTests/EIP155", "/TransactionTestsFiller/EIP155", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttTransactionTestEip155VCheck)
-{
-	dev::test::executeTests("ttTransactionTestVRule", "/TransactionTests/EIP155", "/TransactionTestsFiller/EIP155", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttTransactionTestEip155)
-{
-	dev::test::executeTests("ttTransactionTest", "/TransactionTests/EIP155", "/TransactionTestsFiller/EIP155", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttTransactionTestEip155VitaliksTestsHomestead)
-{
-	dev::test::executeTests("ttTransactionTestEip155VitaliksTests", "/TransactionTests/Homestead", "/TransactionTestsFiller/Homestead", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttTransactionTestHomestead)
+class transactiontestfixture
 {
-	dev::test::executeTests("ttTransactionTest", "/TransactionTests/Homestead", "/TransactionTestsFiller/Homestead", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttTransactionTest)
-{
-	dev::test::executeTests("ttTransactionTest", "/TransactionTests", "/TransactionTestsFiller", dev::test::doTransactionTests);
-}
-
-BOOST_AUTO_TEST_CASE(ttWrongRLPTransactionHomestead)
-{
-	std::string fillersPath =  dev::test::getTestPath() + "/src/TransactionTestsFiller/Homestead";
-	if (!dev::test::Options::get().filltests)
-		dev::test::executeTests("ttWrongRLPTransaction", "/TransactionTests", "/TransactionTestsFiller/Homestead", dev::test::doTransactionTests);
-	else
+public:
+	transactiontestfixture()
 	{
-		dev::test::TestOutputHelper::initTest();
-		dev::test::copyFile(fillersPath + "/ttWrongRLPTransaction.json", dev::test::getTestPath() + "/TransactionTests/Homestead/ttWrongRLPTransaction.json");
-	}
-}
+		string casename = boost::unit_test::framework::current_test_case().p_name;
 
-BOOST_AUTO_TEST_CASE(ttWrongRLPTransaction)
-{
-	std::string fillersPath = dev::test::getTestPath() + "/src/TransactionTestsFiller";
-	if (!dev::test::Options::get().filltests)
-		dev::test::executeTests("ttWrongRLPTransaction", "/TransactionTests", "/TransactionTestsFiller", dev::test::doTransactionTests);
-	else
-	{
-		dev::test::TestOutputHelper::initTest();
-		dev::test::copyFile(fillersPath + "/ttWrongRLPTransaction.json", dev::test::getTestPath() + "/TransactionTests/ttWrongRLPTransaction.json");
+		if ((casename == "ttWrongRLPFrontier" || casename == "ttWrongRLPHomestead") && test::Options::get().filltests)
+			copyAllFilesFromFolder(casename);
+		else
```

Comment:
The execution reaches `fillAllFilesInFolder` even when `test::Options::get().filltests` is false.  Is this correct?

Reply:
yes. it seems this function name is a bit misleading afterall. although the same name is used in other tests

Reply:
I see.  I'll rename the function in another PR.

Reply:
we are going to remove those functions. no need to another PR

Reply:
I'm not sure if I will like your changes, so I'll proceed with the changes I like for certain.